src/content/css-geometry-tools.js

```diff
@@ -1,57 +1,51 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import DOMRect from './dom-rect';
+// Imported interfaces incorrectly throw an error in eslint:
+// https://github.com/benmosher/eslint-plugin-import/issues/726
```

Comment:
not only interfaces actually, that's true for all types that aren't real classes. OK, maybe that's a definition for interfaces :D